<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderItem</class>
 <widget class="QWidget" name="OrderItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="orderID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="isFinished">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="orderInfo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>331</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>35</y>
     <width>131</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>View Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="isFinishedIcon">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
